src/main/resources/data/categories/Returns.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Name of the file to be consistent with other filenames should start with lowercase `returns.json`

Reply:
Done